src/Language/PureScript/Interactive/Parser.hs

```diff
@@ -33,16 +33,16 @@ parseDotFile filePath s = first show $ do
 -- |
 -- Parses PSCI metacommands or expressions input from the user.
 --
-parseCommand :: String -> Either String Command
+parseCommand :: String -> Either String [Command]
 parseCommand cmdString =
   case cmdString of
     (':' : cmd) -> parseDirective cmd
     _ -> parseRest psciCommand cmdString
 
-parseRest :: P.TokenParser a -> String -> Either String a
+parseRest :: P.TokenParser a -> String -> Either String [a]
```

Comment:
This PR all looks great but I’m just slightly concerned about this change; I’m not sure it makes sense with all of the use sites. In particular should the places it’s used in parseDirective be changing? Unless I’m missing something, `:browse Prelude Control.Bind` will now queue two Browse commands, which seems a little odd to me.

Reply:
Ah yes, this was unintentional.  Thanks!